config/config.go

```diff
@@ -200,15 +201,16 @@ type TiKVClient struct {
 }
 
 var defaultConf = Config{
-	Host:        "0.0.0.0",
-	Port:        4000,
-	Store:       "mocktikv",
-	Path:        "/tmp/tidb",
-	RunDDL:      true,
-	SplitTable:  true,
-	Lease:       "10s",
-	TokenLimit:  1000,
-	EnableChunk: true,
+	Host:            "0.0.0.0",
+	Port:            4000,
+	Store:           "mocktikv",
+	Path:            "/tmp/tidb",
+	RunDDL:          true,
+	SplitTable:      true,
+	Lease:           "10s",
+	TokenLimit:      1000,
+	EnableChunk:     true,
+	EnableStreaming: false,
```

Comment:
Can we make it as a session variable? Thus we can compare the results between streaming and normal APIs  without restarting tidb-server.